### ISSUE TITLE
Missing files and model

### ISSUE THREAD

**Issue by JonathanSamelson** [gautamkumarjaiswal/Ground-Truth-Generation#1]
Hi !

First, congratulations for your work, I read your article and I was really interested by this project.
I would have liked to try by myself but there are some files missing (such as images, train folders...) but most importantly the .h5 model to test with a random image. I don't know if it was deliberately removed from your project, but I hope you could share it so we can try out directly.

Thank you in advance,

Jonathan

**Comment by gautamkumarjaiswal** [comment]
Thank you @JonathanSamelson. You can download the dataset from [here](efrosgans.eecs.berkeley.edu/pix2pix/datasets/maps.tar.gz) and unzip it to the current folder. After unzipping operation you will get a folder named 'maps' inside this folder, you will find to subfolder 'train' and 'val' which contains training set and test set. Now you are ready to train model from sketch.
If you want to use pre trained model, please download .h5 model from [here](https://drive.google.com/drive/folders/1SqyYkTeDyibVQ49WVsdF3Mi_0UtqbPQ-?usp=sharing). After downloading pre-trained model you can directly test it using 'testDBimage.py'. I trained with 30 epoch on Linux GPU Titan V. Please let me know if you find any difficulty.

**Comment by isilayy** [comment]
Hi I dont find dataset this link.
